src/App.jsx

```diff
@@ -1,33 +1,36 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
+let todoList = [
+  {
+    id: 1,
+    title: "Eat breakfast"
+  },
+  {
+    id: 2,
+    title: "Brush teeth"
+  },
+  {
+    id: 3,
+    title: "shower"
```

Comment:
nitpick: Consistency is key, so I'd either expect that this `shower` title is capitalized or all of the above titles are left uncapitalized.